octoprint_mrbeam/scripts/update_script.py

```diff
@@ -212,10 +214,10 @@ def install_wheels(install_queue):
             "install",
             "--disable-pip-version-check",
             "--upgrade",  # Upgrade all specified packages to the newest available version. The handling of dependencies depends on the upgrade-strategy used.
-            "--force-reinstall",  # Reinstall all packages even if they are already up-to-date.
```

Comment:
Could it happen that a package gets damaged and we can't reinstall?

Reply:
I checked and it seemed that we needed this to install a v0.13.0 for example over the developer 0+unknown version